src/styles/globals.css

```diff
@@ -63,6 +63,19 @@
   .container {
     @apply px-[3rem] md:px-["4rem"] lg:px-["6rem"] xl:px-["8rem"] mx-auto;
   }
+
```

Comment:
if this classname are not used across multiple component, remove them and do the styling per component